src/core/tables.js

```diff
@@ -0,0 +1,163 @@
+// @ts-check
+// Module core/tables
+// Handles tables in the document.
+// Generates a Table of Tables wherever there is a #tot element.
+
+import {
+  addId,
+  getIntlData,
+  renameElement,
+  showWarning,
+  wrapInner,
+} from "./utils.js";
+import { html } from "./import-maps.js";
+
+export const name = "core/tables";
+
+const localizationStrings = {
+  en: {
+    list_of_tables: "List of Tables",
+    table: "Table ",
+  },
+  ja: {
+    table: "図 ",
+    list_of_tables: "図のリスト",
+  },
+  ko: {
+    table: "그림 ",
+    list_of_tables: "그림 목록",
+  },
+  nl: {
+    table: "Figuur ",
+    list_of_tables: "Lijst met figuren",
+  },
+  es: {
+    table: "Figura ",
+    list_of_tables: "Lista de Figuras",
+  },
+  zh: {
+    table: "图 ",
+    list_of_tables: "规范中包含的图",
+  },
+  de: {
+    table: "Abbildung",
+    list_of_tables: "Abbildungsverzeichnis",
+  },
```

Comment:
We can add the translations later as we learn more languages 😄

Reply:
Ahh got ya.
I misunderstood that these were perhaps languages where we had a translator available.
But it sounds more like when a translation is requested we find a translator.

Reply:
I've removed the anchor-to-table translation test. Let me know if there is a good way to preserve it. I suppose I could keep it in English?